.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -356,7 +360,7 @@ stages:
         target: builder
         baseImage: $(nativeBaseImage)
         useNativeSdkVersion: $(useNativeSdkVersion)
-        command: "BuildNativeTracerHome"
+        command: "BuildNativeTracerHome BuildNativeLoader RunTracerNativeTests RunNativeLoaderNativeTests"
```

Comment:
So we're moving the native loader build to the _tracer_ home - given tracer home currently takes longer to build than profiler home, this will impact overall throughput... do we _need_ to move it? 🤔

Reply:
(Discussed offline, this is only temporary until we have the universal binary, so it's probably fine)